lib/internal/Magento/Framework/CompiledInterception/Test/Integration/CompiledInterceptor/_out_interceptors/Item.txt

```diff
@@ -0,0 +1,87 @@
+
+use Magento\Framework\Config\ScopeInterface;
+use Magento\Framework\ObjectManagerInterface;
+
+/**
+ * Interceptor class for @see \Magento\Framework\CompiledInterception\Test\Integration\CompiledInterceptor\Custom\Module\Model\Item
+ */
+class Interceptor extends \Magento\Framework\CompiledInterception\Test\Integration\CompiledInterceptor\Custom\Module\Model\Item
+{
+    /**
+     * @var ScopeInterface
+     */
+    private $____scope = null;
+
+    /**
+     * @var ObjectManagerInterface
+     */
+    private $____om = null;
+
+    /**
+     * @var \Magento\Framework\CompiledInterception\Test\Integration\CompiledInterceptor\Custom\Module\Model\ItemPlugin\Simple
+     */
+    private $____plugin_simple_plugin = null;
+
+    /**
+     * @var \Magento\Framework\CompiledInterception\Test\Integration\CompiledInterceptor\Custom\Module\Model\ItemPlugin\Advanced
+     */
+    private $____plugin_advanced_plugin = null;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(\Magento\Framework\ObjectManagerInterface $____om, \Magento\Framework\Config\ScopeInterface $____scope)
+    {
+        $this->____om = $____om;
+        $this->____scope = $____scope;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        switch ($this->____scope->getCurrentScope()) {
+        	case 'global':
+        	case 'emptyscope':
+        		$result = parent::getName();
+        		
+        		return (($tmp = $this->____plugin_simple_plugin()->afterGetName($this, $result)) !== null) ? $tmp : $result;
+        	default:
+        		$this->____plugin_advanced_plugin()->beforeGetName($this);
+        		
+        		$result = $this->____plugin_advanced_plugin()->aroundGetName($this, function(){
+        			$result = parent::getName();
+        			
+        			return (($tmp = $this->____plugin_simple_plugin()->afterGetName($this, $result)) !== null) ? $tmp : $result;
+        		});
+        		
+        		return (($tmp = $this->____plugin_advanced_plugin()->afterGetName($this, $result)) !== null) ? $tmp : $result;
+        }
+    }
+
+    /**
+     * plugin "simple_plugin"
+     * @return \Magento\Framework\CompiledInterception\Test\Integration\CompiledInterceptor\Custom\Module\Model\ItemPlugin\Simple
+     */
+    private function ____plugin_simple_plugin()
+    {
+        if ($this->____plugin_simple_plugin === null) {
+        	$this->____plugin_simple_plugin = $this->____om->get(\Magento\Framework\CompiledInterception\Test\Integration\CompiledInterceptor\Custom\Module\Model\ItemPlugin\Simple::class);
+        }
+        return $this->____plugin_simple_plugin;
+    }
+
+    /**
+     * plugin "advanced_plugin"
+     * @return \Magento\Framework\CompiledInterception\Test\Integration\CompiledInterceptor\Custom\Module\Model\ItemPlugin\Advanced
+     */
+    private function ____plugin_advanced_plugin()
+    {
+        if ($this->____plugin_advanced_plugin === null) {
```

Comment:
To avoid issues with state modification by plugins in a stateful mode like Magento + Swoole, we need to remove class level caching for plugins references via private properties. So instead of `$this->____plugin_advanced_plugin = $this->____om->get(ClassNamespace::class)` it should be `$____plugin_advanced_plugin = $this->____om->get(ClassNamespace::class)`. All plugins references should be local variables.